cmake/install.cmake

```diff
@@ -6,6 +6,9 @@ foreach(_library
   libprotoc)
   set_property(TARGET ${_library}
     PROPERTY INTERFACE_INCLUDE_DIRECTORIES
+    $<BUILD_INTERFACE:${protobuf_BINARY_DIR}>
```

Comment:
the export command is never called though. Adding support for it would be valuable, but should be addressed in a separate PR.